docs/design.md

```diff
@@ -4,7 +4,8 @@ by Spencer Kimball from early 2014.
 
 # Overview
 
-Cockroach is a distributed key:value datastore which supports **ACID
+Cockroach is a distributed key:value datastore (SQL and structured
+data layers of cockroach have yet to be defined) which supports **ACID
```

Comment:
nit/question: do we need to manually wrap in docs? my editor wraps markdown automatically and i think github does as well.